README.md

```diff
@@ -253,7 +254,230 @@ if ('delta' in quote) {
+if ('delta' in quote) {
+  const deltaPrice = quote.delta;
+}
+
+// if used mode: 'delta'
+const deltaPrice = quote.delta;
+```
+
+
+### 2. Approve srcToken for DeltaContract
+
+```ts
+const approveTxHash = await simpleSDK.delta.approveTokenForDelta(amount, Token1);
+```
+
+Alternatively sign Permit (DAI or Permit1) or Permit2 TransferFrom with DeltaContract as the verifyingContract
+
+```ts
+const DeltaContract = await simpleSDK.delta.getDeltaContract();
+
+// values depend on the Permit type and the srcToken
+const signature = await signer._signTypedData(domain, types, message);
+```
+
+See more on accepted Permit variants in [ParaSwap documentation](https://developers.paraswap.network/api/paraswap-delta/build-and-sign-a-delta-order#supported-permits)
+
+
+### 3. Sign and submit a Delta Order
+
+```ts
+// calculate acceptable destAmount
+const slippagePercent = 0.5;
+  const destAmountAfterSlippage = (
+    +deltaPrice.destAmount *
+    (1 - slippagePercent / 100)
+  ).toString(10);
+
+const signableOrderData = await simpleSDK.delta.buildDeltaOrder({
+  deltaPrice,
+  owner: account,
+  // beneficiary: anotherAccount, // if need to send the output destToken to another account
+  // permit: "0x1234...", // if signed a Permit1 or Permit2 TransferFrom for DeltaContract
+  srcToken: Token1,
+  destToken: Token2,
+  srcAmount: amount,
+  destAmount: destAmountAfterSlippage, // minimum acceptable destAmount
+  // partner: "..." // if available
+});
+
+const signature = await simpleSDK.delta.signDeltaOrder(signableOrderData);
+
+const deltaAuction = await simpleSDK.delta.postDeltaOrder({
+  // partner: "..." // if available
+  // partiallyFillabel: true, // allow the Order to be partially filled as opposed to fill-or-kill
+  order: signableOrderData.data,
+  signature,
+});
+```
+
+#### 3.a.
+
+As an option the `buildDeltaOrder + signDeltaOrder + signDeltaOrder` can be combined into one SDK call with the following code
+
+```ts
+const deltaAuction = await simpleSDK.delta.submitDeltaOrder({
+  deltaPrice,
+  owner: account,
+  // beneficiary: anotherAccount, // if need to send output destToken to another account
+  // permit: "0x1234...", // if signed a Permit1 or Permit2 TransferFrom for DeltaContract
+  // partiallyFillabel: true, // allow the Order to be partially filled as opposed to fill-or-kill
+  srcToken: Token1,
+  destToken: Token2,
+  srcAmount: amount,
+  destAmount: destAmountAfterSlippage, // minimum acceptable destAmount
+});
+```
+
+This allows to simplify the flow at the expense of control over the Order signing.
+
+#### 3.b adding partner fee
+
+A portion of destToken will be collected as a partner fee if `partner` parameter is provided to `buildDeltaOrder` (and `submitDeltaOrder`). The `partnerFee` itself is `deltaPrice.partnerFee`
+
+To examine the default partnerFee parameters (`{partnerAddress: Address, partnerFee: number, takeSurplus: boolean}`), you can call. These parameters are then encoded in Order.partnerAndFee field.
```

Comment:
I can call ...?

Reply:
Who you gonna call? :ghost: :no_entry_sign: